### ISSUE TITLE
improve weather widget

### ISSUE THREAD

**Issue by apaleslimghost** [apaleslimghost/almanac#104]
i'm thinking design our own icons or commission somebody to do it. the new icons should show biome, weather & day/night/moon phase independently. maybe could support dream-date's multiple moons?

**Comment by rowanmanning** [comment]
Yeah that whole section would be nice as an image rather than have anything written 👍 imagine it's a fair few icons required though.

- 7 moon phases (8 if there's a specific icon for new moon)
- X many biomes (could be a lot, I've just gone down [this Wikipedia hole](https://en.wikipedia.org/wiki/Biome#Classifications)), do we also display something slightly different if you're in a city/town in that biome?
- 18 weather types at the moment

That said I like the idea of doing this with SVGs that can be composed together, that'd be a useful service that could even run separately :D

**Comment by apaleslimghost** [comment]
what i was thinking is we could generate inline svg from react and compose sub icons

**Comment by apaleslimghost** [comment]
we should probably do #20 first so we have a better idea of the weather model
